src/kademlia/node.cpp

```diff
@@ -270,7 +270,7 @@ void node::incoming(aux::listen_socket_handle const& s, msg const& m)
 		return;
 	}
 
-	char y = *(y_ent.string_ptr());
+	const char y = *(y_ent.string_ptr());
```

Comment:
I prefer the const to the right of the type it applies to, i.e. [continental const placement](https://kuhllib.com/2012/01/17/continental-const-placement/).